x-pack/plugins/monitoring/public/components/apm/apm_metrics.tsx

```diff
@@ -41,6 +46,7 @@ interface Props {
       container: boolean;
     };
   };
+  StatusComponent: React.FC<{ stats: Stats }>;
```

Comment:
Nit: you could define these as `StatusComponentProps` for clarity
```suggestion
  StatusComponent: React.FC<StatusComponentProps>;
```

Reply:
It's a fair point. I'm going to leave it for now just because this is the only place it's used, and with it being the generic portion of the `React.FC` type I think the intention is clear.

There's a few of these `Status` components so I feel the best solution is probably some common location for the type they can all share, but it's an easy way to go down a rabbit hole 😅